gammapy/analysis/tests/test_analysis.py

```diff
@@ -318,10 +318,10 @@ def test_analysis_ring_background():
     analysis.get_excess_map()
     assert isinstance(analysis.datasets[0], MapDataset)
     assert_allclose(
-        analysis.datasets[0].npred_background().data[0, 10, 10], 0.091799, rtol=1e-2
+        analysis.datasets[0].npred_background().data[0, 10, 10], 0.096767, rtol=1e-2
```

Comment:
because the default cutout mode in the DatasetsMaker is "partial" instead of "trim", not sure why it makes a difference. (I don't see an option to change the cutout_mode in the configuration for now)

Reply:
Maybe keep the same cutout mode for now, to make sure the results are equivalent? Then in a follow up PR the mode could be changed if there is a better default...

Reply:
ok changed the default to trim.
Then I noticed that the changes also come from the stacking of the MapDatasetOnOff into the MapDataset after the RingBackground because I forgot the conversion .to_map_dataset(). This should return an error no ?